nixos/modules/services/web-apps/immich.nix

```diff
@@ -110,6 +111,37 @@ in
       description = "The group immich should run as.";
     };
 
+    settings = mkOption {
+      default = null;
```

Comment:
Should we remove the default so people have to actively choose against declarative configuration?
cc @mweinelt who has opinions about this stemming from Home Assistant

Reply:
Do you want to force everyone to make a choice here or have it declarative-by-default?

Reply:
Declarative by default is problematic for people who already configured their instance through the web interface.